.gitignore

```diff
@@ -10,3 +10,4 @@ app_static/
 settings_local.py
 
 *.swp
+.drone.sec.yml
```

Comment:
A .drone.sec yml is allowed to reach git while the source file should not.
Tried to run locally but the credentials were wrong, both from my version and another branch.